src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -179,8 +185,9 @@ public function buildView(FormView $view, FormInterface $form, array $options)
             : $this->createChoiceListView($choiceList, $options);
 
         $view->vars = array_replace($view->vars, array(
+            'widget' => $options['widget'],
             'multiple' => $options['multiple'],
-            'expanded' => $options['expanded'],
+            'expanded' => $options['expanded'], // BC
```

Comment:
IMHO should be marked as internal since it's still used in the implementation of this class